Formula/d/dwarfs.rb

```diff
@@ -0,0 +1,119 @@
+class Dwarfs < Formula
+  desc "Fast high compression read-only file system for Linux, Windows, and macOS"
+  homepage "https://github.com/mhx/dwarfs"
+  url "https://github.com/mhx/dwarfs/releases/download/v0.10.1/dwarfs-0.10.1.tar.xz"
+  sha256 "db785e0e0f257fa4363d90153db34127add4552791a72998b30ded787840d039"
+  license "GPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "googletest" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "brotli"
+  depends_on "double-conversion"
+  depends_on "flac"
+  depends_on "fmt"
+  depends_on "gflags"
+  depends_on "glog"
+  depends_on "howard-hinnant-date"
+  depends_on "libarchive"
+  depends_on "libevent"
+  depends_on "libsodium"
+  depends_on "llvm" if OS.mac? && DevelopmentTools.clang_build_version < 1500
+  depends_on "lz4"
+  depends_on "nlohmann-json"
+  depends_on "openssl@3"
+  depends_on "parallel-hashmap"
+  depends_on "range-v3"
+  depends_on "utf8cpp"
+  depends_on "xxhash"
+  depends_on "xz"
+  depends_on "zstd"
+
+  on_linux do
+    depends_on "jemalloc"
+  end
+
+  fails_with :clang do
+    build 1400
+    cause "Not all required C++20 features are supported"
+  end
+
+  fails_with :gcc do
+    version "11"
+    cause "Not all required C++20 features are supported"
+  end
+
+  def install
+    args = std_cmake_args + %W[
```

Comment:
```suggestion
    args = %W[
```
Pass `std_cmake_args` after the `args` array. See below.

Reply:
Done.